### ISSUE TITLE
Is this project discontinued / unmaintained?

### ISSUE THREAD

**Issue by MonaMayrhofer** [bbodi/notecalc3#45]
I saw that this project seems to have not received much attention for the last 2 years, which is a shame, cause it looks very promising.

Is this project officially discontinued or is someone planning to get back to development whenever they are ready?

Thanks a lot!

**Comment by bbodi** [comment]
Thanks for the nice words :)  

The project is not dead, though the development pace has decreased dramatically, and I stopped publishing the code into this public repository because I got some shady emails back then, and I started to being paranoid that somebody will steal my work.

The next release will be out this year for sure, and it already contains tremendous fixes and new features.

Feel free to watch this repository, as I will publish the changelog here.
